articles/machine-learning/how-to-create-attach-kubernetes.md

```diff
@@ -416,6 +422,21 @@ kubectl get secret/azuremlfessl -o yaml
 >[!Note]
 >Kubernetes stores the secrets in base-64 encoded format. You will need to base-64 decode the `cert.pem` and `key.pem` components of the secrets prior to providing them to `attach_config.enable_ssl`. 
 
+### Delete azureml-fe related resources
```

Comment:
how about add this section after line 440 which tells customer how to connect customer by kubectl?